client/src/app/shared/shared-forms/select/select-languages.component.ts

```diff
@@ -23,6 +23,7 @@ export class SelectLanguagesComponent implements ControlValueAccessor, OnInit {
   availableLanguages: SelectOptionsItem[] = []
 
   allLanguagesGroup = $localize`All languages`
+  allLanguagesGroupOrder = 1
```

Comment:
You don't need to create a variable here, prefer to put `1` line `52` so we can easily identify the difference between the two groups